XLS-0080d-permissioned-domains/README.md

```diff
@@ -0,0 +1,190 @@
+* Creating a `PermissionedDomainDelete` transaction.
+
+This feature will require an amendment. Since this feature doesn't bring any functionality to the XRPL by itself, it will be gated by other amendments that use it instead.
+
+### 1.1. Terminology
+
+* **Domain**: A collection of rules indicating what accounts may be a part of it. This spec includes credential-gating, but more options could be added in the future. A domain is focused on "who can participate". The "how they can participate" (e.g. what tokens they may use) aspect can be done in other ways in the primitives they're used.
+* **Domain Rules**: The set of rules that govern a domain, i.e. the credentials it accepts.
+* **Domain Owner**: The account that created a domain, and is the only one that can modify its rules or delete it.
+* **Domain Member**: An account that satisfies the rules of the domain (i.e. has one of the credentials that are accepted by the domain). There is no explicit joining step; as long as the account has a valid credential, it is a member.
+
+## 2. On-Ledger Object: `PermissionedDomain`
+
+This object represents a permissioned domain.
+
+### 2.1. Fields
+
+| Field Name | Required? | JSON Type | Internal Type | Description |
+|------------|-----------|-----------|---------------|-------------|
+|`LedgerIndex`|✔️|`string`|`Hash256`|The unique ID of the ledger object.|
+|`Flags`| ✔️|`number`|`UInt32`|Flag values associated with this object.|
+|`LedgerEntryType`|✔️|`string`|`UInt16`|The ledger object's type (`PermissionedDomain`).|
+|`Owner`|✔️|`string`|`AccountID`|The account that controls the settings of the domain.|
+|`Sequence`|✔️|`number`|`UInt32`|The `Sequence` value of the `PermissionedDomainSet` transaction that created this domain. Used in combination with the `Account` to identify this domain.|
+|`AcceptedCredentials`|✔️|`array`|`STArray`|The credentials that are accepted by the domain. Ownership of one of these credentials automatically makes you a member of the domain.|
+|`PreviousTxnID`|✔️|`string`|`Hash256`|The identifying hash of the transaction that most recently modified this entry.|
+|`PreviousTxnLgrSeq`|✔️|`number`|`UInt32`|The ledger index that contains the transaction that most recently modified this object.|
+
+#### 2.1.1. `LedgerIndex`
+
+The ID of this object will be a hash that incorporates the `Owner` and `Sequence` fields, combined with a unique space key for `PermissionedDomain` objects, which will be defined during implementation.
+
+This value will be used wherever a `DomainID` is required.
+
+#### 2.1.2. `AcceptedCredentials`
+
+This is an array of inner objects referencing a type of credential. The maximum length of this array is 10.
+
+The array will be sorted by `Issuer`, so that searching it for a match is more performant.
+
+If this field is not included in the domain, then credentials do not need to be used when interacting with anything using the domain (i.e. there are no credential-based rules).
```

Comment:
This is confusing to me - the above table shows `AcceptedCredentials` to be required, but the wording here implies that it is optional.

Reply:
The paragraph was left in from when it was optional. Removed.

Reply:
I believe this is from extended spec when Domain will operate by other rules as well. But for now it is limited by credentials only.